2015-08-20 15:04:55 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:05:11 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:05:11 - DEBUG --> GFSalesforce::export_feed: Starting export for entry #4 to Salesforce... 
2015-08-20 15:05:11 - DEBUG --> GFSalesforce::create: Starting the create process... 
2015-08-20 15:05:11 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:05:11 - DEBUG --> process_merge_vars(): Starting... 
2015-08-20 15:05:11 - DEBUG --> process_merge_vars(): Completed. 
2015-08-20 15:05:11 - DEBUG --> DA::filter_api_merge_vars() - Starting adding DA data to merge vars. 
2015-08-20 15:05:11 - DEBUG --> DA::filter_api_merge_vars() - Not a Lead object. 
2015-08-20 15:05:11 - DEBUG --> GFSalesforce::Instance: init singleton instance 
2015-08-20 15:05:11 - DEBUG --> GFSalesforce::create: Upserting using primary field of `Email` 
2015-08-20 15:05:11 - DEBUG --> GFSalesforce::create: Upserting 
2015-08-20 15:05:12 - ERROR --> GFSalesforce::create:

Exception While Exporting Entry
There was an error exporting Entry #4 for Form #1. Here's the error:
'2015-08-20 19:05:11' is not a valid value for the type xsd:dateTime 
2015-08-20 15:05:12 - DEBUG --> GFSalesforce::export_feed: Contact failed to be created/updated 
2015-08-20 15:07:01 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:07:01 - DEBUG --> GFSalesforce::export_feed: Starting export for entry #4 to Salesforce... 
2015-08-20 15:07:01 - DEBUG --> GFSalesforce::create: Starting the create process... 
2015-08-20 15:07:01 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:07:01 - DEBUG --> process_merge_vars(): Starting... 
2015-08-20 15:07:01 - DEBUG --> process_merge_vars(): Completed. 
2015-08-20 15:07:01 - DEBUG --> DA::filter_api_merge_vars() - Starting adding DA data to merge vars. 
2015-08-20 15:07:01 - DEBUG --> DA::filter_api_merge_vars() - Not a Lead object. 
2015-08-20 15:07:01 - DEBUG --> GFSalesforce::Instance: init singleton instance 
2015-08-20 15:07:01 - DEBUG --> GFSalesforce::create: Upserting using primary field of `Email` 
2015-08-20 15:07:01 - DEBUG --> GFSalesforce::create: Upserting 
2015-08-20 15:07:01 - ERROR --> GFSalesforce::create:

Exception While Exporting Entry
There was an error exporting Entry #4 for Form #1. Here's the error:
'2015-08-20 19:05:11' is not a valid value for the type xsd:dateTime 
2015-08-20 15:07:01 - DEBUG --> GFSalesforce::export_feed: Contact failed to be created/updated 
2015-08-20 15:07:39 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:07:39 - DEBUG --> GFSalesforce::export_feed: Starting export for entry #3 to Salesforce... 
2015-08-20 15:07:39 - DEBUG --> GFSalesforce::create: Starting the create process... 
2015-08-20 15:07:39 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:07:39 - DEBUG --> process_merge_vars(): Starting... 
2015-08-20 15:07:39 - DEBUG --> process_merge_vars(): Completed. 
2015-08-20 15:07:39 - DEBUG --> DA::filter_api_merge_vars() - Starting adding DA data to merge vars. 
2015-08-20 15:07:39 - DEBUG --> DA::filter_api_merge_vars() - Not a Lead object. 
2015-08-20 15:07:39 - DEBUG --> GFSalesforce::Instance: init singleton instance 
2015-08-20 15:07:39 - DEBUG --> GFSalesforce::create: Upserting using primary field of `Email` 
2015-08-20 15:07:39 - DEBUG --> GFSalesforce::create: Upserting 
2015-08-20 15:07:39 - ERROR --> GFSalesforce::create:

Exception While Exporting Entry
There was an error exporting Entry #3 for Form #1. Here's the error:
'2015-08-20 18:49:59' is not a valid value for the type xsd:dateTime 
2015-08-20 15:07:39 - DEBUG --> GFSalesforce::export_feed: Contact failed to be created/updated 
2015-08-20 15:08:02 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:09:49 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:09:56 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:09:58 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:09:58 - DEBUG --> getFields: fields have been cached. 
2015-08-20 15:09:58 - DEBUG --> getFields: fields have been cached. 
2015-08-20 15:10:05 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:05 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:10:05 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:05 - DEBUG --> getFields: fields are being fetched for $objectType Lead 
2015-08-20 15:10:05 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:05 - DEBUG --> getFields: fields are being fetched for $objectType Lead 
2015-08-20 15:10:08 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:08 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:10:08 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:08 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:10:08 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:08 - DEBUG --> getFields: fields are being fetched for $objectType Contact 
2015-08-20 15:10:09 - DEBUG --> getFields: fields have been cached. 
2015-08-20 15:10:16 - DEBUG --> getFields: fields have been cached. 
2015-08-20 15:10:17 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:17 - DEBUG --> getFields: fields have been cached. 
2015-08-20 15:10:18 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:18 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:10:18 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:18 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:10:18 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:18 - DEBUG --> getFields: fields are being fetched for $objectType Contact 
2015-08-20 15:10:19 - DEBUG --> getFields: fields have been cached. 
2015-08-20 15:10:20 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:20 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:10:20 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:20 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:10:20 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:10:20 - DEBUG --> getFields: fields are being fetched for $objectType Contact 
2015-08-20 15:10:20 - DEBUG --> getFields: fields have been cached. 
2015-08-20 15:11:03 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:11:03 - DEBUG --> getFields: fields have been cached. 
2015-08-20 15:11:03 - DEBUG --> getFields: fields have been cached. 
2015-08-20 15:11:18 - DEBUG --> api_is_valid(): valid api 
2015-08-20 15:11:18 - DEBUG --> GFSalesforce::export_feed: Starting export for entry #1 to Salesforce... 
2015-08-20 15:11:18 - DEBUG --> GFSalesforce::create: Starting the create process... 
2015-08-20 15:11:18 - DEBUG --> get_api(): API connection already set. 
2015-08-20 15:11:18 - DEBUG --> process_merge_vars(): Starting... 
2015-08-20 15:11:18 - DEBUG --> process_merge_vars(): Completed. 
2015-08-20 15:11:18 - DEBUG --> DA::filter_api_merge_vars() - Starting adding DA data to merge vars. 
2015-08-20 15:11:18 - DEBUG --> DA::filter_api_merge_vars() - Not a Lead object. 
2015-08-20 15:11:18 - DEBUG --> GFSalesforce::Instance: init singleton instance 
2015-08-20 15:11:18 - DEBUG --> GFSalesforce::create: Upserting using primary field of `Email` 
2015-08-20 15:11:18 - DEBUG --> GFSalesforce::create: Upserting 
2015-08-20 15:11:18 - ERROR --> GFSalesforce::create:

Exception While Exporting Entry
There was an error exporting Entry #1 for Form #1. Here's the error:
'2015-08-20 18:45:15' is not a valid value for the type xsd:dateTime 
2015-08-20 15:11:18 - DEBUG --> GFSalesforce::export_feed: Contact failed to be created/updated 
2015-08-20 15:13:27 - ERROR --> GFSalesforce::create:

Exception While Exporting Entry
There was an error exporting Entry #5 for Form #1. Here's the error:
'2015-08-20 19:13:26' is not a valid value for the type xsd:dateTime 
2015-08-20 15:13:39 - ERROR --> GFSalesforce::create:

Exception While Exporting Entry
There was an error exporting Entry #5 for Form #1. Here's the error:
'2015-08-20 19:13:26' is not a valid value for the type xsd:dateTime 
2015-08-20 15:15:16 - ERROR --> GFSalesforce::create: There was an error exporting Entry #5 for Form #1. Salesforce responded with:
stdClass Object
(
    [fields] => Array
        (
            [0] => Id
        )

    [message] => Contact ID: id value of incorrect type: John;Walker
    [statusCode] => MALFORMED_ID
)
 
2015-08-20 15:16:04 - ERROR --> GFSalesforce::create: There was an error exporting Entry #5 for Form #1. Salesforce responded with:
stdClass Object
(
    [fields] => Array
        (
            [0] => X18_Digit_Contact_ID__c
        )

    [message] => Unable to create/update fields: X18_Digit_Contact_ID__c. Please check the security settings of this field and verify that it is read/write for your profile or permission set.
    [statusCode] => INVALID_FIELD_FOR_INSERT_UPDATE
)
 
2015-08-20 15:16:08 - ERROR --> GFSalesforce::create: There was an error exporting Entry #5 for Form #1. Salesforce responded with:
stdClass Object
(
    [fields] => Array
        (
            [0] => X18_Digit_Contact_ID__c
        )

    [message] => Unable to create/update fields: X18_Digit_Contact_ID__c. Please check the security settings of this field and verify that it is read/write for your profile or permission set.
    [statusCode] => INVALID_FIELD_FOR_INSERT_UPDATE
)
 
